<launch>
	<node name="replanner" type="tensor_replanner" pkg="tensorPlanner" output="screen" >
		<remap from="/cmd_vel" to="/private/nav/cmd_vel"/>
	</node>
	<node pkg="nifti_teleop" type="priority_wrapper.py"
		name="priority_wrapper_tensor">
	</node>
	<node name="marker" type="goal_interact_marker" pkg="tensorPlanner" output="screen" />
</launch>
